Composer/packages/types/src/telemetry.ts

```diff
@@ -294,3 +295,13 @@ export type TelemetryClient = {
  * however, they are only logged to Application Insights after the user opts in to data collection.
  */
 export const persistedEvents: TelemetryEventName[] = ['SessionStarted', 'HandoffToComposerCompleted'];
+
+/**
+ * These events are ones which are always tracked regardless of whether tracking is opted
+ * into or not (i.e. those having to do with turning telemetry itself on or off). If
+ * the user opts out, we will strip all PII from the event before sending it.
+ */
+export const alwaysTrackEvents: TelemetryEventName[] = ['TelemetryOptInOut'];
+
+/** Names of the properties containing PII Data */
+export const piiProperties = ['userId', 'rootProjectId', 'projectId'];
```

Comment:
RootProjectId and projectId are just session GUIDS. They shouldn't be classified as PII

Reply:
@garypretty Should we consider project ids PII?